### PR TITLE
Bpk 3006 update colours

### PR DESCRIPTION
**Pull request by tiagoengel**
TODO:
- [x] Test iOS

+ [x] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack-react-native/blob/master/CONTRIBUTING.md)

Remember to include the following changes:
+ [x] `UNRELEASED.md`
+ [x] `README.md`
+ [x] Tests
+ [x] Docs (either update [backpack-docs](https://github.com/Skyscanner/backpack-docs) now, or create a follow up ticket)

_If you are curious about how we review, please read through the [code review guidelines](https://github.com/Skyscanner/backpack/blob/master/CODE_REVIEW_GUIDELINES.md)_

### CODE REVIEW

File: packages/react-native-bpk-component-badge/src/BpkBadge.js [hunk 1]
```diff
@@ -30,9 +30,9 @@ import {
 import {
   colorGray900,
   colorGray50,
```
Comment (georgegillams) [inline packages/react-native-bpk-component-badge/src/BpkBadge.js]: all the grays need changing too, but it's not a visual change so lets do that in another PR 👍

File: packages/react-native-bpk-component-progress/src/__snapshots__/BpkProgress-test.android.js.snap [hunk 2]
```diff
@@ -39,7 +39,7 @@ exports[`Android BpkProgress should not go over the max bound 1`] = `
   style={
     Array [
       Object {
-        "backgroundColor": "rgb(223, 247, 236)",
+        "backgroundColor": undefined,
```
Comment (georgegillams) [inline packages/react-native-bpk-component-progress/src/__snapshots__/BpkProgress-test.android.js.snap]: 🤔

File: packages/react-native-bpk-theming/stories.js [hunk 3]
```diff
@@ -199,12 +196,12 @@ class BpkThemePicker extends Component<{}, State> {
 
     this.themes = {
       blue: generateThemeAttributes(
-        colorBlue400,
-        colorBlue500,
+        colorSkyBlue,
```
Comment (georgegillams) [inline packages/react-native-bpk-theming/stories.js]: Given that this is a theming test, shouldn't we pick two different colours to ensure we can see it working?

Comment (tiagoengel) [reply packages/react-native-bpk-theming/stories.js]: I guess, I removed the gradient because the new brand shouldn't have any gradients anymore.

File: packages/react-native-bpk-component-progress/src/__snapshots__/BpkProgress-test.android.js.snap (hunk 2 above)
Comment (tiagoengel) [reply packages/react-native-bpk-component-progress/src/__snapshots__/BpkProgress-test.android.js.snap]: Good catch, was using `sagano` instead of `segano` 🤦‍♂

Comment (tiagoengel) [reply packages/react-native-bpk-component-progress/src/__snapshots__/BpkProgress-test.android.js.snap]: And I just realised that was the right name and the token name was wrong :/

File: packages/react-native-bpk-theming/stories.js (hunk 3 above)
Comment (georgegillams) [reply packages/react-native-bpk-theming/stories.js]: We haven't yet made a decision about that. We might still support it for partners

File: packages/react-native-bpk-component-progress/src/__snapshots__/BpkProgress-test.android.js.snap (hunk 2 above)
Comment (georgegillams) [reply packages/react-native-bpk-component-progress/src/__snapshots__/BpkProgress-test.android.js.snap]: Oh yeah we fixed that. I think if you get a later version of bpk-tokens it should be ok

File: packages/react-native-bpk-component-flat-list/src/BpkFlatListItem.ios.js [hunk 4]
```diff
@@ -59,13 +59,13 @@ const styles = StyleSheet.create({
     flex: 1,
   },
   textSelected: {
-    color: colorBlue600,
+    color: colorSkyBlue,
```
Comment (ojcurtis) [inline packages/react-native-bpk-component-flat-list/src/BpkFlatListItem.ios.js]: We should check if this blue should be Sky Blue and not another Blue

File: packages/react-native-bpk-component-horizontal-nav/src/BpkHorizontalNavItem.js [hunk 5]
```diff
@@ -57,7 +57,7 @@ const styles = StyleSheet.create({
     color: colorGray300,
   },
   textSelected: {
-    color: colorBlue700,
+    color: colorSkyBlue,
```
Comment (ojcurtis) [inline packages/react-native-bpk-component-horizontal-nav/src/BpkHorizontalNavItem.js]: We should check if this blue should be Sky Blue and not another Blue

File: packages/react-native-bpk-component-horizontal-nav/src/BpkHorizontalNavSelectedIndicator.js [hunk 6]
```diff
@@ -27,15 +27,15 @@ import {
...
+  colorSkyBlue,
   borderSizeLg,
 } from 'bpk-tokens/tokens/base.react.native';
 
 import { REQUIRED_THEME_ATTRIBUTES, themePropType } from './theming';
 
 const styles = StyleSheet.create({
   selectedIndicator: {
-    backgroundColor: colorBlue700,
+    backgroundColor: colorSkyBlue,
```
Comment (ojcurtis) [inline packages/react-native-bpk-component-horizontal-nav/src/BpkHorizontalNavSelectedIndicator.js]: We should check if this blue should be Sky Blue and not another Blue

File: packages/react-native-bpk-component-text/stories.js [hunk 7]
```diff
@@ -109,28 +109,28 @@ storiesOf('react-native-bpk-component-text', module)
   ))
   .add('Colours', () => (
     <View>
-      <BpkText textStyle="xxxl" style={{ color: colorBlue700 }}>
+      <BpkText textStyle="xxxl" style={{ color: colorSkyBlue }}>
```
Comment (ojcurtis) [inline packages/react-native-bpk-component-text/stories.js]: We should check if this blue should be Sky Blue and not another Blue

Comment (tiagoengel) [reply packages/react-native-bpk-component-text/stories.js]: this is just stories in the example app, I don't think it really matters

Comment (ojcurtis) [reply packages/react-native-bpk-component-text/stories.js]: That is true!

File: packages/react-native-bpk-component-horizontal-nav/src/BpkHorizontalNavSelectedIndicator.js (hunk 6 above)
Comment (tiagoengel) [reply packages/react-native-bpk-component-horizontal-nav/src/BpkHorizontalNavSelectedIndicator.js]: This is what we have for Android currently. I'll check with Will

File: packages/react-native-bpk-component-horizontal-nav/src/BpkHorizontalNavItem.js (hunk 5 above)
Comment (tiagoengel) [reply packages/react-native-bpk-component-horizontal-nav/src/BpkHorizontalNavItem.js]: This is what we have for Android currently, I'll check with Will.

Comment (tiagoengel) [reply packages/react-native-bpk-component-horizontal-nav/src/BpkHorizontalNavItem.js]: All good from Will :)

Comment (ojcurtis) [reply packages/react-native-bpk-component-horizontal-nav/src/BpkHorizontalNavItem.js]: Sweet! In which case will update my review!
